2-improved.cpp

```diff
@@ -0,0 +1,13 @@
+#include <stdio.h>
+
+int sum(num1, num2)
```

Comment:
Each parameter should consist of a type followed by an identifier `int sum(int num1, int num2)`